lark/parser_frontends.py

```diff
@@ -182,14 +180,14 @@ def match(self, term, text, index=0):
 
 
 def create_earley_parser__dynamic(lexer_conf, parser_conf, options=None, **kw):
-        earley_matcher = EarleyRegexpMatcher(lexer_conf)
-        return xearley.Parser(parser_conf, earley_matcher.match, ignore=lexer_conf.ignore, **kw)
+    earley_matcher = EarleyRegexpMatcher(lexer_conf)
+    return xearley.Parser(lexer_conf, parser_conf, earley_matcher.match, ignore=lexer_conf.ignore, **kw)
```

Comment:
If the parser takes `lexer_conf`, we don't need to explicitly provide `lexer_conf.ignore`.